taichi/system/timer.cpp

```diff
@@ -57,18 +57,82 @@ double Time::get_time() {
 }
 #endif
 
+#ifdef _WIN64
+#include <Windows.h>
+#pragma comment(lib, "Winmm.lib")
```

Comment:
> I have added `win_usleep` , `win_msleep` for different use cases on Windows .
> None behavior changes on Linux. You can try to run on your machine to check if anything is different : )

Nice work! I can confirm that it's running happily on my machine!
One concern, what is `Winmm.lib`? Will it cause compat issue?
We'd better include it in `CMakeList.txt` instead of an inline `#pragma comment`, WDYT?
I'd like to provide more usage about linking libraries in CMake if you are not familiar to.

Reply:
```Winmm.lib``` is just a system multimedia API , which contains high resolution timer event functions .
```Winmm.dll``` can found in system path . And the minimum support client is winXP . This might not cause incompatibility problem.
See : [MSDN](https://docs.microsoft.com/en-us/previous-versions/dd757634(v=vs.85))

At the last commit  I have moved ```#pragma commit``` away and used ```target_link_library``` instead .